packages/@react-spectrum/provider/src/Provider.tsx

```diff
@@ -152,6 +152,19 @@ const ProviderWrapper = React.forwardRef(function ProviderWrapper(props: Provide
     colorScheme: props.colorScheme ?? Object.keys(theme).filter(k => k === 'light' || k === 'dark').join(' ')
   };
 
+  let hasWarned = useRef(false);
+  useEffect(() => {
```

Comment:
do we want to exclude this code if `process.env.NODE_ENV` is production? seems like useful dev runtime warning but not something we want to include once bundled.

Reply:
hmm... so the stuff inside the if will get removed no?
but a hook can't be behind the if, so it has to stay?
trying to look this up in annoying, i keep getting irrelevant hits

Reply:
hmm right. could be that we wrap the inside? im not sure of the implementation, just that it doesn't seem to make sense to be on production builds (probably?)

Reply:
o, lol! I did !== 'development' and definitely meant !== 'production'

Reply:
hmm are we sure we want to do this? I think it may come up only in production sometimes. e.g. components as a service, when things are integrated together only in prod?

Reply:
I'm fine either way, i just thought this was what we'd agreed on yesterday
happy to turn it on all the time as well, i do agree it's more likely to happen in production, at least for groups like document cloud

Reply:
Let's have it included in production. The most obvious use case for this to appear in would be when deployed within another app.